package.json

```
@@ -1,5 +1,6 @@
 {
   "type": "module",
+  "packageManager": "pnpm@10.0.0",
   "engines": {
     "node": ">= 20.0.0"
```

```suggestion
    "node": ">= 20.0.0",
    "pnpm": ">=10.0.0"
```

https://pnpm.io/package_json#engines